react-props-and-expressions/my-app/src/Components/CustomButton.jsx

```diff
@@ -0,0 +1,17 @@
+export default function Buttons() {
+  return (
+    <div className="flex center">
+      <CustomButton text="I" className="red" />
+      <CustomButton text="Know" className="blue" />
+      <CustomButton text="React" className="green" />
+    </div>
+  );
+}
```

Comment:
This works, but I wouldn't go about doing it this way. `CustomButton` should only be `return`ing one instance of our `button`. You should be creating the structure where you would call the component instead rather then where you create the component.

So this structure should be inside of `App.jsx`